experimental/GITFans/GITFans.jl

```diff
@@ -481,7 +481,7 @@ function hashes_to_polyhedral_fan(orbit_list, hash_list, hom)
                       for rays in rays_maxcones]
 
     # the indices of rays that belong to each repres. cone (0-based)
-    index_result_cones = [sort([findfirst(x -> x == v, allrays)-1
+    index_result_cones = [sort([findfirst(x -> x == v, allrays)
```

Comment:
After this change, the comment `(0-based)` above the code line is no longer true.

Reply:
Thanks, I had actually removed this but it didn't make it into the last commit.